core/domain/question_jobs_one_off_test.py

```diff
@@ -19,8 +19,9 @@
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
 import ast
+import datetime
 
-from core.domain import question_jobs_one_off
+from core.domain import question_jobs_one_off as question_jobs
```

Comment:
Probably better not to use an alias. Where lines are too long, break by inserting parens (I gave an example below).

Reply:
Done